config/locales/en.yml

```diff
@@ -295,7 +304,7 @@ en:
       hint: "A dependant is a child or adult who:"
       lives_with: lives with your client
       depends: depends on your client for financial support
-      no_partner: Don't include your client's partner if they have one
+      no_partner: Do not include your client’s partner as a dependant.
```

Comment:
Is the intention here to only show this if the client has a partner? (Because without the 'if' clause, this sentence looks strongly like it doesn't belong when there is no partner.)

Reply:
Yes this is only shown if the client has a partner IIRC

Reply:
Looking at the view file, it doesn't have any conditional logic in it yet - because previously the "if" in the wording meant we could just about get away without it. Now that we don't have the "if" any more, I think we need to add in the conditional logic.

Reply:
I think the challenge here is that I'm working on 2 tickets with dependants. The other one does have 'if' logic - I've updated the content here for now pending the dependants ticket that is being reworked anyway

Reply:
Ohhh, I see. Fair enough!